pandas/io/sas/sasreader.py

```diff
@@ -29,8 +29,11 @@ def read_sas(filepath_or_buffer, format=None, index=None, encoding=None,
     DataFrame if iterator=False and chunksize=None, else SAS7BDATReader
     or XportReader
     """
-
     if format is None:
+        buffer_error_msg = "If this is a buffer object rather\
+                than a string name, you must specify a format string"
+        if not isinstance(filepath_or_buffer,compat.string_types):
```

Comment:
You have some PEP8 errors (we check for them in the travis build, that is the reason travis failed). For example, here there should be a space after the comma.

Here is the full list (you can find it at the bottom of the failing test (third travis build), but I also recommend setting up your IDE to check for this):
```
pandas/io/sas/sasreader.py:6:1: E302 expected 2 blank lines, found 1
pandas/io/sas/sasreader.py:35:45: E231 missing whitespace after ','
pandas/io/tests/sas/test_sas.py:5:1: E302 expected 2 blank lines, found 1
```